pod_project/pods/migrations/0004_AlterFields.py

```diff
@@ -43,7 +44,25 @@ class Migration(migrations.Migration):
         migrations.AlterField(
             model_name='pod',
             name='is_draft',
-            field=models.BooleanField(default=True, help_text='If you check this box, the video will be visible and accessible only by you.', verbose_name='Draft'),
```

Comment:
J'ai lancé un "makemigrations", qui m'a créé un fichier "0005_auto_xxxx" ; et comme le "migrate" que j'ai effectué ensuite n'a rien modifié je me suis dit que je pouvais incorporer le contenu de ce fichier dans “0004_AlterFields.py”, en me disant qu'il porte bien son nom.

![capture d ecran 2015-11-12 a 18 45 48](https://cloud.githubusercontent.com/assets/10742818/11126119/c09caa10-896d-11e5-957e-ae64fe3fa38c.png)

En effet, je serais très surpris qu'il s'agisse de champs de BD :
http://stackoverflow.com/questions/26503826/why-does-django-generate-migrations-for-help-text-and-verbose-name-changes
https://code.djangoproject.com/ticket/21498

Je verrais ça plus tard.
À+
Philippe